packages/console/.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "trailingComma": "all",
+  "singleQuote": true
+}
```

Comment:
This is a change we can and should apply to the entire repository, perhaps piecemeal and then holistically.

Reply:
How do we apply it to the whole monorepo, rather than littering copies into each package?

Reply:
I’m not entirely certain about Prettier, but for Eslint, it was just a matter of moving this file up two directories. Might work. Might not. Otherwise, a symbolic link up two directories in each package.

Reply:
Or, eventually, a package that we can depend upon in all of our projects in any repository.

Reply:
This change should not happen in this PR. Files #465. Resolving.